js/graphicView.js

```diff
@@ -0,0 +1,41 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'core/js/views/componentView';
+
+class GraphicView extends ComponentView {
+
+  preRender() {
+    this.listenTo(Adapt, 'device:changed', this.resizeImage);
+
+    this.checkIfResetOnRevisit();
+  }
+
+  postRender() {
+    this.resizeImage(Adapt.device.screenSize, true);
+  }
+
+  checkIfResetOnRevisit() {
+    const isResetOnRevisit = this.model.get('_isResetOnRevisit');
+
+    if (isResetOnRevisit) {
+      this.model.reset(isResetOnRevisit);
+    }
+  }
+
+  resizeImage(width, setupInView) {
```

Comment:
This function is no longer needed as we can do this in the template. React views will rerender on `device:changed` (see event attachment line 7).

Move this bit into `postRender`:
```js
    this.$('.graphic__widget').imageready(() => {
      this.setReadyStatus();
      this.setupInviewCompletion('.graphic__widget');
    });
```
Strip out all references to the `resizeImage` function (line 13 and line 7 etc). Remove the `import Adapt` directive.

We'll move this bit into the template:
```js
    const imageWidth = width === 'medium' ? 'small' : width;
    const imageSrc = (this.model.get('_graphic')) ? this.model.get('_graphic')[imageWidth] : '';
    this.$('.js-graphic-set-image-src').attr('src', imageSrc);
```